shared/shot.js

```diff
@@ -374,7 +374,9 @@ class AbstractShot {
     // eslint-disable-next-line no-control-regex
     filenameTitle = filenameTitle.replace(/[:\\<>/!@&?"*.|\x00-\x1F]/g, " ");
     filenameTitle = filenameTitle.replace(/\s{1,4000}/g, " ");
-    let clipFilename = `Screenshot-${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()} ${filenameTitle}`;
+    today = new Date()
```

Comment:
@KopfKrieg note that `date` is the created date of the shot. you'll want to base the filename on that date, as in the line you're replacing